install.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+echo "================================================"
+echo "----------------- Openclassify -----------------"
+echo "----- Automated install script with Docker -----"
+echo "================================================"
+
+if [[ $(which docker) && $(docker --version) ]]; then
+    docker --version
+
+    #if ubuntu install docker
```

Comment:
# If Ubuntu, check if Docker is running

Reply:
it's acutally todo. it's already starting docker if OS is Ubuntu